test/unittests/test_intx_api.cpp

```diff
@@ -259,6 +259,49 @@ TYPED_TEST(uint_api, comparison)
     EXPECT_TRUE(d >= x);
 }
 
+template <typename T>
+struct Wrapper
+{
+    T v;
+
+    friend auto operator<=>(Wrapper, Wrapper) = default;
+};
+
+TYPED_TEST(uint_api, spaceship_operator_with_wrapper)
+{
+    auto a = Wrapper<TypeParam>{TypeParam{1}};
```

Comment:
Please use this syntax (template argument will be deduced).
```suggestion
    Wrapper a{TypeParam{1}};
```

Reply:
Looks like some older compiler are not able to handle it properly. So bring back the template argument

```cpp
Wrapper<TypeParam> a{TypeParam{1}};
```

or try relying on `int` to `TypeParam` conversion:

```cpp
Wrapper<TypeParam> a{1};
```